src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -17,13 +17,12 @@
 package io.delta.tables
 
 import scala.collection.JavaConverters._
-
 import org.apache.spark.sql.delta._
 import io.delta.tables.execution._
 import org.apache.hadoop.fs.Path
-
```

Comment:
hmm, we were trying to follow Apache Spark's coding style as closely as possible here: 
 1. Java imports
 2. Scala imports
 3. 3rd party imports + Delta
 4. Spark imports
Can you remove the new lines to match this specification please?

Reply:
@brkyvz yep! Is there an sbt build step that we could introduce that would check for these import standards?

Reply:
we do have scalastyle checks, but I guess these rules were not added there. I'll see if I can add those so that we don't have to deal with them.